site/assignments/country-catalogue/country-catalogue.rst

```diff
@@ -27,22 +27,44 @@ Part 1 --- Uploading Files to Colab
 Country Class
 =============
 
+The ``Country`` class is simply going to represent an individual country. For our purposes, a ``Country`` will know its
+``name``, ``continent``, ``population``, and land ``area``. In addition to the attributes, a ``Country`` will also be
+able to determine it's population density (``population/area``). A method for determining equality (``__eq__``) and
+generating a nice, human readable string representation of the ``Country`` object (``__repr__``) will be written.
+
 
 Part 2 --- Country Constructor
 ------------------------------
 
+Write the constructor (``__init__``) for the ``Country`` object. The constructor will take a country ``name``, which
+``continent`` it's on, its ``population``, and the total land ``area`` as parameters. The constructor will assign the
+values passed as parameters to their respective attributes --- ``name``, ``continent``, ``population, and ``area``.
+
 
 Part 3 --- Population Density
 -----------------------------
 
+Write a method ``population_density`` that returns the population density of the ``Country`` object. The population
+density should be a float.
+
 
 Part 4 --- Equals and Repr
 --------------------------
 
+Write the ``__eq__`` magic dunder method for the ``Country`` class. For our purposes, two ``Country`` objects will be
```

Comment:
It's _magic method_ or _dunder_. Prefer to use _magic method_ since that is what the python documentation typically refers to them as, though they are officially called [special methods](https://docs.python.org/3/reference/datamodel.html#special-method-names).

Reply:
I intentionally went with "magic dunder" to make sure I piss everyone off. 

Same reason I say Gee Eye Eff for gif. It just pisses everyone off.